pkgs/misc/zeyple/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs, lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "zeyple";
+  version = "cc125b7";
```

Comment:
Can you please request a new release from upstream? It looks like it has been a while...

Reply:
No response from them yet. Could we move forward with this, regardless?